ee/tabby-schema/src/schema/auth.rs

```diff
@@ -278,6 +279,26 @@ pub struct PasswordChangeInput {
     pub new_password2: String,
 }
 
+#[derive(Validate)]
+pub struct UpdateUserNameInput {
+    #[validate(length(
+        min = 2,
+        code = "username",
+        message = "Username must be at least 2 characters"
+    ))]
+    #[validate(length(
+        max = 20,
+        code = "username",
+        message = "Username must be at most 20 characters"
+    ))]
+    #[validate(regex(
+        code = "username",
+        path = "crate::schema::constants::USERNAME_REGEX",
+        message = "Invalid username, only alphabet and space character are allowed, and it must start and end with alphabet"
```

Comment:
```suggestion
        message = "Invalid name, only alphabet and space character are allowed, and it must start and end with alphabet"
```

Is non-english characters, e.j german / chinese valid name with this regex?